invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/header.overrides

```diff
@@ -33,3 +33,9 @@
     font-size: 1rem;
   }
 }
+
+aside.sidebar {
+  h3.hidden {
+    display: none;
```

Comment:
```suggestion
aside.sidebar {
   .hidden {
    display: none;
```
not a blocker: would be enough, especially if you want to hide other elements than h3, but it works for this specific case